locales/en-US/experiments.ftl

```diff
@@ -35,7 +34,7 @@ cliqzToursteps0Copy = Start typing in the URL bar to see real-time results.
 cliqzToursteps1Copy = Open a new tab and add your favorite websites for one-click access.
 cliqzToursteps2Copy = You can always give us feedback or disable Cliqz from Test Pilot.
 containersDescription = Contain yourselves! Create different containers for each of your online lives - your work self, your social self, your personal self - to stay organized and maintain privacy.
-containersIntroduction = Containers let you create profiles in Firefox for all of your online lives. Custom labels and color-coded tabs help keep different activities — like online shopping, travel planning, or checking work email — separate. Because Containers store cookies separately, you can log into the same site with a different account in each Container, and online trackers can’t connect your browsing in one container to another. So you can keep your shopping self separate from your social self from your work self, without worrying about being followed around the Web.
```

Comment:
If you change the content of a localized string, aren't you supposed to change the l10n id too, like 'containersIntroduction2? Otherwise I don't think the localizers know they need to re-localize them.